### PR TITLE
k8s handler uses client data from solar resources

### PR DESCRIPTION
**Pull request by nebril**
~/.kube/config file is not longer used there.

### CODE REVIEW

File: solar_k8s/k8s_handler.py [hunk 1]
```diff
@@ -17,17 +17,14 @@
...
-
-from pykube.config import KubeConfig
-from pykube.http import HTTPClient
 import pykube.objects
-
 import yaml
 
+from solar.core.handlers.base import TempFileHandler
+from solar.core.log import log
+from solar import errors
```
Comment (nhlfr) [inline solar_k8s/k8s_handler.py]: Why did you move imports from solar here? solar is a third-party package in relation to solar_k8s. So keeping them alongside with pyyaml, pykube etc. was good IMO.

Comment (nebril) [inline solar_k8s/k8s_handler.py]: This makes sense.

File: solar_k8s/k8s_handler.py [hunk 2]
```diff
@@ -36,7 +33,7 @@ def __init__(self, resources, handlers=None):
         super(K8S, self).__init__(resources, handlers)
 
     def action(self, resource, action_name):
-        api = HTTPClient(KubeConfig.from_file("~/.kube/config"))
+        api = get_kube_api(resource.transports()[0])
```
Comment (pigmej) [inline solar_k8s/k8s_handler.py]: It would be safer to use "find_named_transport" there.

**Comment by pigmej** [comment]
The same logic should be applied to: https://github.com/loles/solar-k8s-modules/blob/master/solar_k8s/k8s_job_transport.py#L142
